docs/modules/ROOT/pages/how-to-guides/Secure-your-supply-chain/proc_java_dependencies.adoc

```diff
@@ -24,7 +45,43 @@ The JVM build service addressed this concern by allowing you to rebuild your app
 
 * If all the builds are successful, your application's SBOM displays that none of the dependencies came from third-party repositories. However, you may need to troubleshoot when some dependencies fail to build.
 
-.*Procedure*
+.*Procedure Using the CLI*
```

Comment:
Some minor tweaks to the following procedures. Let me know what you think :) 

=== Using the CLI 

Configure the JVM Build Service to automatically setup rebuilds and create a Quay repository to store these rebuilt artifacts. 

.Procedure 

- In the CLI, run `setup rebuilds`.  < source code you provided > 

=== Using the CLI without Quay.io configuration 

If you do not have Quay configuration,  you must manually setup rebuilds. 

.Procedure 

1. Create a file, for example, `config.yaml`. 
2.  In the `config.yaml` file, create a *JBSConfig* resource with the following data: < your provided source yaml > 
3. Log into the build namespace. 
4. Run `kubectl apply -f config.yaml`. 

=== Using the CLI with explicit Quay configuration 

If you want to specify where the rebuilt artifacts are stored, you must use explicit Quay configuration. Also, you must provision a secret to allow the JVM Build Service to push to the repository.

.Procedure

< keep the procedure steps as they currently are >